db/migrate/20240430034154_add_not_null_constraints_and_remove_defaul_values_from_users.rb

```diff
@@ -0,0 +1,15 @@
+class AddNotNullConstraintsAndRemoveDefaulValuesFromUsers < ActiveRecord::Migration[7.1]
+  def change
+    change_column_null :users, :first_name, false
+    change_column_null :users, :last_name, false
+    change_column_null :users, :gender, false
+    change_column_null :users, :date_of_birth, false
+    change_column_null :users, :role, false
+    change_column_null :users, :email, false
+    change_column_null :users, :phone_number, false
+    change_column_null :users, :password_digest, false
+
+    change_column_default :users, :role, from: 2, to: nil
```

Comment:
why are you making users role `patient` to `nil`

Reply:
@kth-wtag Apu, previously the user's role field had a default value of `patient`. as now we are not allowing any default value, that  is why I removed the default value and changed it to `nil`.